Plugins/ReplaceTimestamps/ReplaceTimestamps.plugin.js

```diff
@@ -87,10 +87,11 @@ module.exports = !global.ZeresPluginLibrary
 
             patchMessage() {
                 Patcher.before(DiscordModules.MessageActions, "sendMessage", (_, [, msg]) => {
-                    const regexAGlobal = /(?<!\d)\d{1,2}:\d{2}(?!\d)(am|pm)?/gi;
-                    const regexA = /((?<!\d)\d{1,2}:\d{2}(?!\d))(am|pm)?/i;
+                    const regexAGlobal = /(?<!\d)\d{1,2}:\d{2}(?!\d)(am|pm)?(t|T|d|D|f|F|R)?/gi;
+                    const regexA = /((?<!\d)\d{1,2}:\d{2}(?!\d))(am|pm)?(t|T|d|D|f|F|R)?/i;
                     if (msg.content.search(regexAGlobal) !== -1) msg.content = msg.content.replace(regexAGlobal, x => {
-                        let [, time, mode] = x.match(regexA);
+                        let [, time, mode, format] = x.match(regexA);
+                        this.format = format;
```

Comment:
Why are you assigning it to `this`? Please do something like this:
```suggestion
                        format = format ?? "t";
```